src/components/structures/RoomView.js

```diff
@@ -349,14 +371,14 @@ module.exports = React.createClass({
             // if the event after the one referenced in the read receipt if sent by us, do nothing since
```

Comment:
(I can't leave line comments on expanded sections in github it seems!)

This function starts on `:362` and looks like:

```javascript
onRoomReceipt: function(receiptEvent, room) {
    if (room.roomId == this.props.roomId) {
        // bunch of long stuff
    }
}
```

Can we invert this please to save an indent?

```javascript
onRoomReceipt: function(receiptEvent, room) {
    if (room.roomId !== this.props.roomId) {
        return;
    }
    // bunch of long stuff
}
```